src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -169,36 +163,8 @@ public boolean publish(SlackRequest slackRequest) {
             for (String roomId : roomIds) {
                 HttpPost post;
                 String url;
-                String threadTs = "";
```

Comment:
I don't really understand the purpose of the `roomIds` augmentation above (`// include committer userIds in roomIds`), so I left it alone. Conceivably, it makes more sense in `SlackMessageRequest` now.

Note: in my testing, the `reactions.add` API call only works with the channel id returned by `chat.postMessage`; it fails with a `channel_not_found` error if passed the `roomId` that shows up in this method. So the signature `SlackRequest.getBody(JSONObject roomId)` is a bit odd, but seems right for `chat.postMessage` so I left it this way.

Reply:
when using a thread i think its threadId:channel name

Reply:
I'm resolving this because, with the latest commit, the code still lives in this function and is unchanged from master.